pyop2/sequential.py

```diff
@@ -843,13 +843,12 @@ def compile(self):
         # Blow away everything we don't need any more
         del self._args
         del self._kernel
-        del self._itspace
```

Comment:
You sure don't want to delete `self._iterset` instead?

Reply:
Make sense. Amended.